docs: clarified readme for quarantine script (#16)
trendmicro/cloudone-filestorage-plugins: post-scan-actions/aws-python-promote-or-quarantine/README.md

    - Install the AWS command line interface (CLI). See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
    - Install GNU Make if you don't want to use the AWS CLI. See [GNU Make](https://www.gnu.org/software/make/) for download information.
2. **Create two S3 buckets**

>     - Create a 'Promote bucket' to receive clean files. Example: `FSS-Promote`.

Reviewer: ```suggestion
    - Create a 'Promote bucket' to receive clean files. Example: `fss-promote`.
```
Sorry, miss this example bucket name.

>     - Create a 'Quarantine bucket' to receive quarantined files. Example: `FSS_Quarantine`.

Reviewer: ```suggestion
    - Create a 'Quarantine bucket' to receive quarantined files. Example: `fss-quarantine`.
```
I think underscores and capital letters are invalid for bucket names.
![image](https://user-images.githubusercontent.com/67313382/90225507-16ce9080-de44-11ea-89d4-3019149cc6c4.png)
![image](https://user-images.githubusercontent.com/67313382/90225563-2c43ba80-de44-11ea-8c86-2988f99fd4f6.png)

    - Click **Edit** (on the right).
    - Click **Add environment variable**
        - In the **Key** field, enter `PROMOTEBUCKET` 
        - In the **Value** field, enter `<YOUR_PROMOTE_BUCKET>`
    - Again, click **Add environment variable**
        - In the **Key** field, enter `QUARANTINEBUCKET` 

>         - In the **Value** field, enter `<YOUR_QUARANTINE_BUCKET>` . Example: `FSS_Quarantine`

Reviewer: ```suggestion
        - In the **Value** field, enter `<YOUR_QUARANTINE_BUCKET>` . Example: `fss-quarantine`
```
Sorry, miss this example bucket name.

Reviewer: Hmm, do you miss to change the example name here, @robmaynardjr ?

        - `<YOUR_LAMBDA_FUNCTION_ARN>` is replaced with the Lambda function ARN you found earlier.
    </details>

## Test the Quarantine Function

1. **Download the Eicar test file**
   - The Eicar test virus can be downloaded here: `https://secure.eicar.org/eicar.com`
2. **Upload the Eicar file to the scanning bucket**

    <details>
    <summary>Using AWS Console</summary>

    1. Go to **Services > S3**.
    2. Search for your scanning bucket.
    3. Click on the **Upload** button. 
    4. Find the Eicar test file you downloaded and click **Upload**.
    5. Check your quarantine bucket to see if the Eicar file has moved.
    **NOTE:** It can take about 30-60 seconds or more for the file to move.

    </details>

    <details>
    <summary>Using the AWS CLI</summary>

    - Enter the folowing AWS CLI command to upload the Eicar test file to the scanning bucket:

        `aws s3 cp eicar.com s3://<YOUR_SCANNING_BUCKET> `
    - where:
        - `<YOUR_SCANNING_BUCKET>` is replaced with the S3 scanning bucket name.

> 

Reviewer 2: ```suggestion
    2. Make sure that `eicar.zip` file is present in your Quarantine bucket by listing its contents. Use this command:
        `aws s3 ls s3://<YOUR_QUARANTINE_BUCKET>`
    - where `<YOUR_QUARANTINE_BUCKET>` is replaced with the name of your Quarantine bucket.
    3. Make sure the `eicar` file is no longer present in your ScanningBucket by listing out its contents. Use this command:
        `aws s3 ls s3://<YOUR_SCANNING_BUCKET>
    - where `<YOUR_SCANNING_BUCKET>` is replaced with the name of your ScanningBucket.
```

Author: Suggestions applied. LGTM

>     **NOTE:** It can take about 30-60 seconds or more for the file to move.

Reviewer 2: ```suggestion
    📌 It can take 30-60 seconds or more for the 'move' operation to complete, and during this time, you may see the file in both buckets.
```

Author: Suggestions applied LGTM.
